jsonnet/jsonnet.bzl

```diff
@@ -210,12 +219,10 @@ def _jsonnet_to_json_impl(ctx):
         [
             "set -e;",
             toolchain.jsonnet_path,
-        ] + ["-J %s" % im for im in _add_prefix_to_imports(ctx.label, ctx.attr.imports)] +
-        ["-J %s" % im for im in depinfo.imports.to_list()] + [
-            "-J .",
-            "-J %s" % ctx.genfiles_dir.path,
-            "-J %s" % ctx.bin_dir.path,
```

Comment:
@EdSchouten - the removal of the bin_dir is a divergence in behavior from the past. Do you feel that this is more correct?

(I'm encountering an issue related to this in a large repo - but wanting to understand if the change is intentional)

Reply:
The bindir should still be added for generated source files, through file.root.path.

Reply:
Could you make a small reproduction case?